packages/dds/merge-tree/DEV.md

```diff
@@ -8,3 +8,12 @@ Ths distinction is important, as a removed segment with undefined length may not
 However a not yet visible segment with 0 length may already exist, or will eventually exits on all clients.
 These have implications for eventually consistent conflict resolution. Generally, we ignore removed segments, and special case invisible segments, like in the case
 of conflicting insert as handled in the `breakTie` function
+
+### Zamboni
```

Comment:
@scarlettjlee added some dev notes on zamboni? let please ask questions about this or other topics. trying to make it possible of others to help in this area